### ISSUE TITLE
Cant enter 'add' or 'cancel' --> Plugin doesn't recognize

### ISSUE THREAD

**Issue by YourServerPromo** [Blackvein/Quests#393]
When i try to add a new Stage --> In this example a new location then i have to enter 'add' but the Plugin gives me the message i have to enter it while i have already done that.

Same problem when entering 'cancel'. It always gives me the message to enter something but i already did. And when i try to enter a start region. It thinks 'add' or 'cancel' or some other commands in the Questeditor are regions? Why does the plugin not work suddenly.
My Plugins on the Server are:
http://prntscr.com/c19s1t
Here is the message i am speaking about:
http://prntscr.com/c19s8z

Need ur help very fast.
Thanks a lot :)

**Comment by LeafyMitsuwa** [comment]
Assuming you've got permissions to do so, though whilst unable to provide an explanation for your problem, I'd recommend atleast the following two things whilst waiting on a response.

1. Create a test server, locally, to see if the issue persists with -ONLY- the quests plugin, to rule out interference from any other plugin.
2. Post the version of the quest plugin you are using.

**Comment by YourServerPromo** [comment]
Sorry it was my fault. One of my admins has translated the commands in the language file as well. Works fine now! Thanks for helping me :)
